libraries/AP_Baro/AP_Baro.cpp

```diff
@@ -136,6 +139,22 @@ const AP_Param::GroupInfo AP_Baro::var_info[] = {
     // Slot 12 used to be TEMP3
 #endif
 
+    // @Param: ADJ_RATE
+    // @DisplayName: baro adjustment rate
+    // @Description: Rate of barometer adjustment using GPS. If enabled (>=0.1), the barometer ground pressure is slowly adjusted with GPS altitude.  Recommended value: 1cm/sec
+    // @Units: cm/s
+    // @Range: 0 10
+    // @Increment: 0.1
+    AP_GROUPINFO("ADJ_RATE", 13, AP_Baro, _gps_adj_step, ADJ_RATE_DEFAULT),
```

Comment:
If it's in a float already as the storage type please just provide this as a meter unit range. Much easier on the code side, and actually long term easier for users. (We've been slowly moving everything to base units when possible)

Reply:
OK, will change